mkosi/initrd.py

```diff
@@ -111,6 +118,12 @@ def main() -> None:
             "--include=mkosi-initrd",
         ]  # fmt: skip
 
+        if args.kernel_image:
+            cmdline += [
+                "--extra-tree",
+                f"{args.kernel_image}:/usr/lib/modules/{args.kernel_version}/vmlinuz",
+            ]
```

Comment:
```suggestion
            cmdline += [
                "--extra-tree", f"{args.kernel_image}:/usr/lib/modules/{args.kernel_version}/vmlinuz",
            ]  # fmt: skip
```
since we usually keep flags and their arguments on the same line.